ourhistory.html

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+---
+
+<section class="jumbotron" style="background-image: url('https://images.ctfassets.net/y3a9myzsdjan/4tq2Pb9tFyEXs6kEexpLK7/cab00ea4d82d21fe461836b3aa9792ba/crossroads-church-old2.jpg'); height: 455px;">
```

Comment:
I'm not crazy about inline styles. Can you think of another way to accomplish this?